docs/resources/identity_role.md

```diff
@@ -45,26 +45,28 @@ EOF
 
 The following arguments are supported:
 
-* `name` - (Required, String) Name of the custom policy.
+* `name` - (Required, String) Specifies the name of the custom policy.
 
-* `description` - (Required, String) Description of the custom policy.
+* `description` - (Required, String) Specifies the description of the custom policy.
 
-* `type` - (Required, String) Display mode. Valid options are *AX*: Account level and *XA*: Project level.
+* `type` - (Required, String) Specifies the display mode of the custom policy. Valid options are as follows:
+  + **AX**: the global service project.
+  + **XA**: region-specific projects.
 
-* `policy` - (Required, String) Document of the custom policy in JSON format. For more details, please refer to the
-  [official document](https://support.huaweicloud.com/intl/en-us/usermanual-iam/iam_01_0017.html).
+* `policy` - (Required, String) Specifies the content of the custom policy in JSON format. For more details,
+  please refer to the [official document](https://support.huaweicloud.com/intl/en-us/usermanual-iam/iam_01_0017.html).
 
 ## Attribute Reference
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The role id.
+* `id` - The custom policy ID.
 
 * `references` - The number of references.
 
 ## Import
 
-Roles can be imported using the `id`, e.g.
+IAM custom policies can be imported using the `id`, e.g.
 
 ```
```

Comment:
```suggestion
```bash
```

Reply:
it doesn't matter when the ID is a string without "/", however, I will add it.